base/src/main/java/com/thoughtworks/go/util/SystemEnvironment.java

```diff
@@ -214,6 +214,8 @@
 
     public static GoIntSystemProperty DEPENDENCY_MATERIAL_UPDATE_LISTENERS = new GoIntSystemProperty("dependency.material.check.threads", 3);
 
+    public static GoIntSystemProperty CONFIG_MATERIAL_POST_UPDATE_LISTENERS = new GoIntSystemProperty("config.material.post.update.threads", 3);
```

Comment:
@maheshp in the summary, you written this default was supposed to be `2`. But here we have 3.

Reply:
Oops, will change it to 2.